frontend/talentsearch/src/js/components/Browse/OngoingRecruitmentSection.tsx

```diff
@@ -0,0 +1,1191 @@
+              id: "v0/leo",
+              description: "Description for the 'IT-04 Manager' classification",
+            }),
+            poolAdvertisementId: selectPoolIdForSection(
+              pools,
+              PoolStream.SoftwareSolutions,
+              "IT",
+              4,
+            ),
+            applyMessage: intl.formatMessage(
+              {
+                defaultMessage: "Apply to level {number}",
+                id: "oKc30q",
+                description: "Invitation to apply at a certain level number",
+              },
+              { number: 4 },
+            ),
+          },
+        ],
+      },
+    ],
+  };
+
+  return (
+    <>
+      <Heading
+        level="h2"
+        Icon={CpuChipIcon}
+        color="purple"
+        data-h2-margin="base(0, 0, x1, 0)"
+      >
+        {intl.formatMessage({
+          defaultMessage: "Apply to ongoing recruitment",
+          id: "RBsGRp",
+          description: "title for section with ongoing pool advertisements",
+        })}
+      </Heading>
+      <p data-h2-margin="base(0, 0, x1, 0)" data-h2-font-weight="base(700)">
+        {intl.formatMessage({
+          id: "2czUAZ",
+          defaultMessage:
+            "Not seeing an active recruitment that matches your skillset? No problem, we still want to hear from you.",
+          description: "summary for section with ongoing pool advertisements",
+        })}
+      </p>
+      <p>
+        {intl.formatMessage({
+          id: "BWvfeG",
+          defaultMessage:
+            "We also offer passive recruitment buckets that allow us to find talent fast when the demand arises. While there’s no guarantee a job will result from the opportunities below, it’s an easy way for your name and resume to be found by managers when the time comes. Feel free to submit your name to any bucket that matches your skills.",
+          description:
+            "instructions for section with ongoing pool advertisements",
+        })}
+      </p>
+      <div data-h2-padding="base(x2, 0, 0, 0) p-tablet(x3, 0, 0, 0)">
+        <Accordion.Root
+          data-h2-margin="base(0)"
+          data-h2-padding="base(0)"
+          type="multiple"
+        >
+          {dataModel.streams
+            .filter(
+              // filter streams with no classifications with a pool ID to apply to
+              (stream) =>
+                !!stream.classifications.find(
+                  (classification) => classification.poolAdvertisementId,
+                ),
+            )
+            .map((stream) => (
+              <Accordion.Item value={stream.key} key={stream.key}>
+                <Accordion.Trigger
+                  subtitle={stream.summary}
+                  data-h2-font-size="base(h4)"
+                >
+                  {stream.title}
+                </Accordion.Trigger>
```

Comment:
This should use the `headerAs` prop to render an `<h3 />` instead of `<h2 />`.

```suggestion
                <Accordion.Trigger
                  subtitle={stream.summary}
                  data-h2-font-size="base(h4)"
                  headerAs="h3"
                >
                  {stream.title}
                </Accordion.Trigger>
```

Reply:
Thanks!  I just downloaded the Wave plugin so I'll try to stay better on top of this!
Fixed in [fix headings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5043/commits/2c7be8d62a59a75ee3df90fda881f7d406b3346e)